tests/test_series.py

```diff
@@ -681,8 +709,8 @@ def test_types_scalar_arithmetic() -> None:
     res_pow3: pd.Series = s.pow(0.5)
```

Comment:
Why didn't you convert these as well?

Reply:
Because if I convert them it would have to hint at `pd.Series[float]` but right now the type is `pd.Series[Unknown]` so I would rather fix the typehints properly and add the proper check/asert_type when fixed than putting just `pd.Series` (issue that we had in the other PR).

Reply:
OK, that makes sense